cmd/snap/cmd_debug_bootvars_test.go

```diff
@@ -60,3 +61,76 @@ func (s *SnapSuite) TestDebugBootvarsNotOnClassic(c *check.C) {
 	_, err := snap.Parser(snap.Client()).ParseArgs([]string{"debug", "boot-vars"})
 	c.Assert(err, check.ErrorMatches, `the "boot-vars" command is not available on classic systems`)
 }
+
+func (s *SnapSuite) TestDebugSetBootvars(c *check.C) {
+	restore := release.MockOnClassic(false)
+	defer restore()
+	bloader := bootloadertest.Mock("mock", c.MkDir())
+	bootloader.Force(bloader)
+	bloader.BootVars = map[string]string{
+		"snap_mode":       "try",
+		"unrelated":       "thing",
+		"snap_core":       "core18_1.snap",
+		"snap_try_core":   "core18_2.snap",
+		"snap_kernel":     "pc-kernel_3.snap",
+		"snap_try_kernel": "",
+	}
```

Comment:
can we use SetBootVars instead here? 
```suggestion
	err := bloader.SetBootVars(map[string]string{
		"snap_mode":       "try",
		"unrelated":       "thing",
		"snap_core":       "core18_1.snap",
		"snap_try_core":   "core18_2.snap",
		"snap_kernel":     "pc-kernel_3.snap",
		"snap_try_kernel": "",
	})
	c.Assert(err, IsNil)
```